src/main/kotlin/camp/nextstep/edu/step2/StringCalculator.kt

```diff
@@ -10,7 +10,10 @@ package camp.nextstep.edu.step2
  *  StringCalculator().calculate("2 + 3 * 4 / 2") // 10
  */
 
-class StringCalculator {
+object StringCalculator {
+
+    private val leftOneRegex: Regex = Regex("(-?\\d+)")
+    private val rightsRegex: Regex = Regex("([-+/*])\\s?(-?\\d+)")
```

Comment:
```suggestion
    private val leftOneRegex: Regex = Regex("""(-?\d+)""")
    private val rightsRegex: Regex = Regex("""([-+/*])\s?(-?\d+)""")
```
개인적으로 정규식의 이스케이프 때문에 정규식에는 raw string 을 쓰는게 좋은것 같습니다.